tests/rptest/tests/datalake/datalake_e2e_test.py

```diff
@@ -159,9 +159,16 @@ def test_upload_after_external_update(self, cloud_storage_type,
             assert count_after_produce == count, f"{count_after_produce} rows, expected {count}"
 
     @cluster(num_nodes=4)
+    @ignore(catalog_type=CatalogType.NESSIE,
+            cloud_storage_type=CloudStorageType.S3)
+    @ignore(catalog_type=CatalogType.NESSIE,
+            cloud_storage_type=CloudStorageType.ABS)
     @matrix(cloud_storage_type=supported_storage_types(),
             catalog_type=supported_catalog_types())
     def test_remove_expired_snapshots(self, cloud_storage_type, catalog_type):
+        """
+        Nessie doesn't support tags, so it is ignored for this test.
```

Comment:
nit: This comment would be more contextual on the @ignore decorator. Also, it gets printed as test description now in ducktape tests results where it doesn't bear any meaning.